test/host/patch.js

```diff
@@ -28,6 +28,29 @@ describe("host/patch", function() {
 				}
 			});
 		});
+
+		describe("loadFiles()", async function() {
```

Comment:
describe can't take an async function. You will need to either move the setup to an async `before(async function() { ... })` hook, or move the setup inside an async `it("...", async function() { ... })` test.

Reply:
Typo has been fixed, async has been moved into a before hook, found and fixed error with the contents of the export file.